### DISCUSSION TITLE
Fuzzy prompt

### DISCUSSION THREAD

**Discussion by sgarfinkel** [copier-org/copier#1021] [Q&A] [answered]
Does copier have a fuzzy prompt? I have a prompt with a large amount of choices, and I'd like to allow users to type and filter down the available choices.

**Comment by sisp** [comment, marked as answer]
I don't think so. Copier uses [questionary](https://github.com/tmbo/questionary) for prompting and the [select question](https://questionary.readthedocs.io/en/stable/pages/types.html#select) for choices. There's an [open PR for prefix-based search](https://github.com/tmbo/questionary/pull/42), but it looks abandoned. I like the idea of fuzzy choice filtering, but I think questionary would need to support it first, then Copier could use it.
